pkg/indexmanagement/converters.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 func calculateConditions(policy apis.IndexManagementPolicySpec, primaryShards int32) rolloverConditions {
-	// 40GB = 40960 1M messages
-	maxDoc := 40960 * primaryShards
+	// 40GB = 40960 1K messages
+	maxDoc := 40960 * 1000 * primaryShards
```

Comment:
nit: we should likely move these number values into constants

Reply:
If I understand this code correctly we do expose the value of `40960 * 1000 * primaryShards` as "constant" via returned rolloverConditions object. This test verifies it is what it is :-) (not sure how much this specific test is effective atm).

Reply:
May be @vimalk78 can comment on this more. We probably do not have any infra code to really test this or mock it ...

Reply:
the PR comment gives a good description, perhaps we can capture the same as comments in the method. will be useful to read later.

Reply:
its more the fact that we have "arbitrary" numbers. I'm asking for setting const variables for `40960` and `1000`.